src/styles/index.scss

```diff
@@ -1,3 +1,18 @@
-body {
+@import './utils/mixins';
+@import 'variables';
+@import 'header';
+@import 'card';
+@import 'stars';
+@import 'catalog';
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body,
+* {
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser
to read your web document